resources/public/api/conf/3.0/schemas/retrieve/metadata.json

```
@@ -1,71 +1,153 @@
+          },
+          {
+            "enum": [
+              "updatedLossEvent"
+            ],
+            "description": "The calculation was triggered internally by HMRC on receipt of an updated pre-MTD brought forward loss made by the customer.\nThis event is only applicable for a \"finalDeclaration\" calculation type. This event is not currently supported and will be supported in the future."
+          },
+          {
+            "enum": [
+              "newAnnualAdjustmentEvent"
+            ],
+            "description": "The calculation was triggered internally by HMRC on receipt of a request for a new adjustable summary calculation by the customer. This event is not currently supported."
+          },
+          {
+            "enum": [
+              "updatedAnnualAdjustmentEvent"
+            ],
+            "description": "The calculation was triggered internally by HMRC on receipt of an adjustment to an adjustable summary calculation made by the customer. This event is not currently supported."
+          },
+          {
+            "enum": [
+              "unattendedCalculation"
+            ],
+            "description": "HMRC has updated the calculation for you. You can see more details in your record-keeping software."
+          }
+        ]
+      },
+      "calculationTimestamp": {
+        "description": "The timestamp of when the calculation was performed in ISO 8601 UTC format:",
+        "type": "string",
+        "format": "YYYY-MM-DDThh:mm:ssZ",
+        "example": "2019-02-15T09:35:15:094Z"
+      },
+      "calculationType": {
+        "description": "The type of calculation performed. Limited to the following possible values:",
+        "type": "string",
+        "enum": [
+          "inYear - Cannot be used for a final declaration.",
+          "finalDeclaration - Can be used when submitting a final declaration."
+        ]
+      },
+      "intentToSubmitFinalDeclaration": {
+        "description": "A boolean to indicate whether the calculation can be used to make a final declaration. The value must be true or false.",
+        "type": "boolean",
+        "exaample": "true"
+      },
+      "finalDeclaration": {
+        "description": "A boolean to indicate whether the calculation has been used to make a final declaration. The value must be true or false.",
+        "type": "boolean",
+        "exaample": "false"
+      },
+      "finalDeclarationTimestamp": {
+        "description": "The timestamp of when the calculation was performed in ISO 8601 UTC format:",
+        "type": "string",
+        "format": "YYYY-MM-DDThh:mm:ssZ",
+        "example": "2019-02-15T09:35:15:094Z"
+      },
+      "periodFrom": {
+        "description": "This defines the earliest date from which the calculation was performed. This will be the accounting period start date (for self-employments) or the tax year start date, whichever occurs first for the income sources supplied. Must conform to the format:",
```

The "format" string does not get rendered. Maybe remove it and add the "full-date" id as below: 
```suggestion
        "id": "full-date",
        "description": "This defines the earliest date from which the calculation was performed. This will be the accounting period start date (for self-employments) or the tax year start date, whichever occurs first for the income sources supplied.",
```